frontend/src/components/UserContainer.tsx

```diff
@@ -0,0 +1,212 @@
+import { useEffect, useState } from "react";
+import { AppService, HelloWorld, User, UserService } from "../api";
+import { IonButton, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle, IonInput, IonItem, IonLabel, IonList } from "@ionic/react";
+
+interface ContainerProps { }
+
+const UserContianer: React.FC<ContainerProps> = () => {
```

Comment:
typo: "UserContainer"

Reply:
Renamed now. Thank you for keeping eyes open.